src/main/java/org/sonarsource/auth/github/UserIdentityFactory.java

```diff
@@ -44,12 +44,12 @@ public UserIdentityFactory(GitHubSettings settings) {
     this.settings = settings;
   }
 
-  public UserIdentity create(GsonUser gson, @Nullable List<GsonTeams.GsonTeam> teams) {
+  public UserIdentity create(GsonUser user, @Nullable String email, @Nullable List<GsonTeams.GsonTeam> teams) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUndocumentedApi)